skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -297,14 +344,87 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+*Changes in "-Filter" parameter in Teams PowerShell version 3.0.0 and above*
+- Assigned Plan filter: Previous format will no longer be supported. Existing filters like `AssignedPlan eq '<some-xml-string>'` will stop working. This will need to be modified to the below format:
```

Comment:
```suggestion
- Assigned Plan filter: Previous format will no longer be supported. Existing filters like `AssignedPlan eq '<some-xml-string>'` will stop working. This will need to be modified to the following format:
```